basic/src/server/index.js

```diff
@@ -45,22 +44,20 @@ server
             }
             return acc;
         }, {productIds: [], collectionIds: [], shouldFetchStore: false});
-        const componentsStateSsrManager = new ComponentsStateSsrManager(POS_ID, 'prod')
-        const context = {};
+        stateBuilder.buildState(dataToFetch.productIds, dataToFetch.collectionIds, dataToFetch.shouldFetchStore).then((initialState) => {
```

Comment:
I think a better format here would be object format:
```
stateBuilder.buildState({productIds: dataToFetch.productIds, collectionIds: dataToFetch.collectionIds, shouldFetchStore: dataToFetch .shouldFetchStore})
```

Each of those properties would be optional (but at least one required).

It's much easier to read when there are "named properties" because it's not easy to memorize arguments order.